Resources/doc/usage.rst

```diff
@@ -15,11 +15,28 @@ Add these two lines in your layout:
         <script src="<?php echo $view['assets']->getUrl('bundles/fosjsrouting/js/router.js') ?>"></script>
         <script src="<?php echo $view['router']->generate('fos_js_routing_js', array('callback' => 'fos.Router.setData')) ?>"></script>
 
-.. note::
-
-    If you are not using Twig, then it is no problem. What you need is to add
+.. note:: If you are not using Twig, then it is no problem. What you need is to add
```

Comment:
this change looks wrong to me.

Reply:
How so? All I did was remove two line-breaks?

Reply:
and this is precisely the issue, as having some content on the same line than the directive or as an indented block following it has a totally different meaning in the rST syntax.

Reply:
Ok - I must be being miss-lead by the editor I'm using - the only difference I see is that without the line-breaks the text of the note is closer to the heading of the note (and therefore to my eye looks neater ;-). I've reverted the change in a soon-to-be pushed commit.